diff -urN kopete-0.10.3.orig/kopete/kopete/addcontactwizard/addcontactwizard_base.ui kopete-0.10.3/kopete/kopete/addcontactwizard/addcontactwizard_base.ui
--- kopete-0.10.3.orig/kopete/kopete/addcontactwizard/addcontactwizard_base.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/addcontactwizard/addcontactwizard_base.ui	2005-11-03 00:00:22.000000000 +0000
@@ -558,9 +558,4 @@
     <tabstop>protocolListView</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/addcontactwizard/fastaddcontactwizard_base.ui kopete-0.10.3/kopete/kopete/addcontactwizard/fastaddcontactwizard_base.ui
--- kopete-0.10.3.orig/kopete/kopete/addcontactwizard/fastaddcontactwizard_base.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/addcontactwizard/fastaddcontactwizard_base.ui	2005-11-03 00:00:22.000000000 +0000
@@ -213,7 +213,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/config/accounts/addaccountwizardpage2.ui kopete-0.10.3/kopete/kopete/config/accounts/addaccountwizardpage2.ui
--- kopete-0.10.3.orig/kopete/kopete/config/accounts/addaccountwizardpage2.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/config/accounts/addaccountwizardpage2.ui	2005-11-03 00:00:22.000000000 +0000
@@ -126,7 +126,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/config/accounts/kopeteaccountconfigbase.ui kopete-0.10.3/kopete/kopete/config/accounts/kopeteaccountconfigbase.ui
--- kopete-0.10.3.orig/kopete/kopete/config/accounts/kopeteaccountconfigbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/config/accounts/kopeteaccountconfigbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -260,9 +260,4 @@
     <tabstop>mButtonRemove</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/config/appearance/appearanceconfig_chatwindow.ui kopete-0.10.3/kopete/kopete/config/appearance/appearanceconfig_chatwindow.ui
--- kopete-0.10.3.orig/kopete/kopete/config/appearance/appearanceconfig_chatwindow.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/config/appearance/appearanceconfig_chatwindow.ui	2005-11-03 00:00:22.000000000 +0000
@@ -314,13 +314,4 @@
     <tabstop>mTransparencyValue</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistbox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kcolorcombo.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/config/appearance/appearanceconfig_colors.ui kopete-0.10.3/kopete/kopete/config/appearance/appearanceconfig_colors.ui
--- kopete-0.10.3.orig/kopete/kopete/config/appearance/appearanceconfig_colors.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/config/appearance/appearanceconfig_colors.ui	2005-11-03 00:00:22.000000000 +0000
@@ -382,16 +382,4 @@
     <tabstop>linkColor</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kfontrequester.h</includehint>
-    <includehint>kfontrequester.h</includehint>
-    <includehint>kfontrequester.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/config/appearance/appearanceconfig_emoticons.ui kopete-0.10.3/kopete/kopete/config/appearance/appearanceconfig_emoticons.ui
--- kopete-0.10.3.orig/kopete/kopete/config/appearance/appearanceconfig_emoticons.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/config/appearance/appearanceconfig_emoticons.ui	2005-11-03 00:00:22.000000000 +0000
@@ -107,8 +107,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistbox.h</includehint>
-    <includehint>ktextedit.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/config/appearance/tooltipeditwidget.ui kopete-0.10.3/kopete/kopete/config/appearance/tooltipeditwidget.ui
--- kopete-0.10.3.orig/kopete/kopete/config/appearance/tooltipeditwidget.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/config/appearance/tooltipeditwidget.ui	2005-11-03 00:00:22.000000000 +0000
@@ -135,8 +135,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/contactlist/customnotifications.ui kopete-0.10.3/kopete/kopete/contactlist/customnotifications.ui
--- kopete-0.10.3.orig/kopete/kopete/contactlist/customnotifications.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/contactlist/customnotifications.ui	2005-11-03 00:00:22.000000000 +0000
@@ -230,9 +230,4 @@
     <tabstop>chkSuppressCommon</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/contactlist/kopetegvipropswidget.ui kopete-0.10.3/kopete/kopete/contactlist/kopetegvipropswidget.ui
--- kopete-0.10.3.orig/kopete/kopete/contactlist/kopetegvipropswidget.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/contactlist/kopetegvipropswidget.ui	2005-11-03 00:00:22.000000000 +0000
@@ -149,8 +149,4 @@
     <tabstop>icnbClosed</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kicondialog.h</includehint>
-    <includehint>kicondialog.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/contactlist/kopetemetalvipropswidget.ui kopete-0.10.3/kopete/kopete/contactlist/kopetemetalvipropswidget.ui
--- kopete-0.10.3.orig/kopete/kopete/contactlist/kopetemetalvipropswidget.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/contactlist/kopetemetalvipropswidget.ui	2005-11-03 00:00:22.000000000 +0000
@@ -537,10 +537,4 @@
     <tabstop>btnExportKABC</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kicondialog.h</includehint>
-    <includehint>kicondialog.h</includehint>
-    <includehint>kicondialog.h</includehint>
-    <includehint>kicondialog.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/kopete/contactlist/linkaddressbookui_base.ui kopete-0.10.3/kopete/kopete/contactlist/linkaddressbookui_base.ui
--- kopete-0.10.3.orig/kopete/kopete/contactlist/linkaddressbookui_base.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/kopete/contactlist/linkaddressbookui_base.ui	2005-11-03 00:00:22.000000000 +0000
@@ -164,9 +164,4 @@
 </includes>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kactivelabel.h</includehint>
-    <includehint>klistview.h</includehint>
-    <includehint>klistviewsearchline.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/libkopete/ui/fileconfirmbase.ui kopete-0.10.3/kopete/libkopete/ui/fileconfirmbase.ui
--- kopete-0.10.3.orig/kopete/libkopete/ui/fileconfirmbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/libkopete/ui/fileconfirmbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -141,11 +141,4 @@
     </grid>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/libkopete/ui/kopetepasswordwidgetbase.ui kopete-0.10.3/kopete/libkopete/ui/kopetepasswordwidgetbase.ui
--- kopete-0.10.3.orig/kopete/libkopete/ui/kopetepasswordwidgetbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/libkopete/ui/kopetepasswordwidgetbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -92,7 +92,4 @@
     <tabstop>mRemembered</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpassdlg.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/alias/aliasdialog.ui kopete-0.10.3/kopete/plugins/alias/aliasdialog.ui
--- kopete-0.10.3.orig/kopete/plugins/alias/aliasdialog.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/alias/aliasdialog.ui	2005-11-03 00:00:22.000000000 +0000
@@ -183,11 +183,4 @@
     <tabstop>kPushButton3</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/alias/aliasdialogbase.ui kopete-0.10.3/kopete/plugins/alias/aliasdialogbase.ui
--- kopete-0.10.3.orig/kopete/plugins/alias/aliasdialogbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/alias/aliasdialogbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -98,10 +98,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/cryptography/cryptographyprefsbase.ui kopete-0.10.3/kopete/plugins/cryptography/cryptographyprefsbase.ui
--- kopete-0.10.3.orig/kopete/plugins/cryptography/cryptographyprefsbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/cryptography/cryptographyprefsbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -190,7 +190,4 @@
     <slot>m_selectOwnKey_stateChanged(int)</slot>
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/highlight/highlightprefsbase.ui kopete-0.10.3/kopete/plugins/highlight/highlightprefsbase.ui
--- kopete-0.10.3.orig/kopete/plugins/highlight/highlightprefsbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/highlight/highlightprefsbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -434,14 +434,4 @@
     <tabstop>m_soundFN</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kcolorbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/history/historyprefsui.ui kopete-0.10.3/kopete/plugins/history/historyprefsui.ui
--- kopete-0.10.3.orig/kopete/plugins/history/historyprefsui.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/history/historyprefsui.ui	2005-11-03 00:00:22.000000000 +0000
@@ -186,9 +186,4 @@
     <tabstop>History_Style</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/latex/latexprefsbase.ui kopete-0.10.3/kopete/plugins/latex/latexprefsbase.ui
--- kopete-0.10.3.orig/kopete/plugins/latex/latexprefsbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/latex/latexprefsbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -159,10 +159,4 @@
     </vbox>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/motionautoaway/motionawayprefs.ui kopete-0.10.3/kopete/plugins/motionautoaway/motionawayprefs.ui
--- kopete-0.10.3.orig/kopete/plugins/motionautoaway/motionawayprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/motionautoaway/motionawayprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -289,9 +289,4 @@
     </vbox>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/netmeeting/netmeetingprefs_ui.ui kopete-0.10.3/kopete/plugins/netmeeting/netmeetingprefs_ui.ui
--- kopete-0.10.3.orig/kopete/plugins/netmeeting/netmeetingprefs_ui.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/netmeeting/netmeetingprefs_ui.ui	2005-11-03 00:00:22.000000000 +0000
@@ -140,9 +140,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kcombobox.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kactivelabel.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/smpppdcs/config/smpppdcsprefs.ui kopete-0.10.3/kopete/plugins/smpppdcs/config/smpppdcsprefs.ui
--- kopete-0.10.3.orig/kopete/plugins/smpppdcs/config/smpppdcsprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/smpppdcs/config/smpppdcsprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -359,12 +359,4 @@
     <tabstop>accountList</tabstop>
 </tabstops>
 <layoutdefaults spacing="0" margin="0"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/statistics/statisticswidget.ui kopete-0.10.3/kopete/plugins/statistics/statisticswidget.ui
--- kopete-0.10.3.orig/kopete/plugins/statistics/statisticswidget.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/statistics/statisticswidget.ui	2005-11-03 00:00:22.000000000 +0000
@@ -237,10 +237,4 @@
     </grid>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kdatepicker.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kdatetbl.h</includehint>
-    <includehint>ktimewidget.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/texteffect/texteffectprefs.ui kopete-0.10.3/kopete/plugins/texteffect/texteffectprefs.ui
--- kopete-0.10.3.orig/kopete/plugins/texteffect/texteffectprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/texteffect/texteffectprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -225,7 +225,4 @@
     <include location="global" impldecl="in implementation">knuminput.h</include>
 </includes>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistbox.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/plugins/webpresence/webpresenceprefs.ui kopete-0.10.3/kopete/plugins/webpresence/webpresenceprefs.ui
--- kopete-0.10.3.orig/kopete/plugins/webpresence/webpresenceprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/plugins/webpresence/webpresenceprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -267,12 +267,4 @@
     <tabstop>includeIMAddress</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/gadu/ui/gaduadd.ui kopete-0.10.3/kopete/protocols/gadu/ui/gaduadd.ui
--- kopete-0.10.3.orig/kopete/protocols/gadu/ui/gaduadd.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/gadu/ui/gaduadd.ui	2005-11-03 00:00:22.000000000 +0000
@@ -354,7 +354,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>krestrictedline.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/gadu/ui/gadueditaccountui.ui kopete-0.10.3/kopete/protocols/gadu/ui/gadueditaccountui.ui
--- kopete-0.10.3.orig/kopete/protocols/gadu/ui/gadueditaccountui.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/gadu/ui/gadueditaccountui.ui	2005-11-03 00:00:22.000000000 +0000
@@ -659,10 +659,4 @@
     <tabstop>dccPort</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>krestrictedline.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/gadu/ui/gaduregisteraccountui.ui kopete-0.10.3/kopete/protocols/gadu/ui/gaduregisteraccountui.ui
--- kopete-0.10.3.orig/kopete/protocols/gadu/ui/gaduregisteraccountui.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/gadu/ui/gaduregisteraccountui.ui	2005-11-03 00:00:22.000000000 +0000
@@ -415,9 +415,4 @@
     <tabstop>valueVerificationSequence</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/gadu/ui/gadusearch.ui kopete-0.10.3/kopete/protocols/gadu/ui/gadusearch.ui
--- kopete-0.10.3.orig/kopete/protocols/gadu/ui/gadusearch.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/gadu/ui/gadusearch.ui	2005-11-03 00:00:22.000000000 +0000
@@ -685,8 +685,4 @@
     <tabstop>listFound</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>krestrictedline.h</includehint>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/groupwise/ui/gwaccountpreferences.ui kopete-0.10.3/kopete/protocols/groupwise/ui/gwaccountpreferences.ui
--- kopete-0.10.3.orig/kopete/protocols/groupwise/ui/gwaccountpreferences.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/groupwise/ui/gwaccountpreferences.ui	2005-11-03 00:00:22.000000000 +0000
@@ -320,7 +320,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/groupwise/ui/gwcontactpropswidget.ui kopete-0.10.3/kopete/protocols/groupwise/ui/gwcontactpropswidget.ui
--- kopete-0.10.3.orig/kopete/protocols/groupwise/ui/gwcontactpropswidget.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/groupwise/ui/gwcontactpropswidget.ui	2005-11-03 00:00:22.000000000 +0000
@@ -205,7 +205,4 @@
     </vbox>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/irc/ui/ircadd.ui kopete-0.10.3/kopete/protocols/irc/ui/ircadd.ui
--- kopete-0.10.3.orig/kopete/protocols/irc/ui/ircadd.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/irc/ui/ircadd.ui	2005-11-03 00:00:22.000000000 +0000
@@ -160,7 +160,4 @@
     <tabstop>tabWidget3</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>mycustomwidget.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/irc/ui/irceditaccount.ui kopete-0.10.3/kopete/protocols/irc/ui/irceditaccount.ui
--- kopete-0.10.3.orig/kopete/protocols/irc/ui/irceditaccount.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/irc/ui/irceditaccount.ui	2005-11-03 00:00:22.000000000 +0000
@@ -1019,9 +1019,4 @@
     <tabstop>addButton</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-    <includehint>klistview.h</includehint>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgaddcontact.ui kopete-0.10.3/kopete/protocols/jabber/ui/dlgaddcontact.ui
--- kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgaddcontact.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/jabber/ui/dlgaddcontact.ui	2005-11-03 00:00:22.000000000 +0000
@@ -97,7 +97,4 @@
     </vbox>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgbrowse.ui kopete-0.10.3/kopete/protocols/jabber/ui/dlgbrowse.ui
--- kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgbrowse.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/jabber/ui/dlgbrowse.ui	2005-11-03 00:00:22.000000000 +0000
@@ -194,8 +194,4 @@
     </connection>
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgjabberchooseserver.ui kopete-0.10.3/kopete/protocols/jabber/ui/dlgjabberchooseserver.ui
--- kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgjabberchooseserver.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/jabber/ui/dlgjabberchooseserver.ui	2005-11-03 00:00:22.000000000 +0000
@@ -101,7 +101,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kactivelabel.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgjabbereditaccountwidget.ui kopete-0.10.3/kopete/protocols/jabber/ui/dlgjabbereditaccountwidget.ui
--- kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgjabbereditaccountwidget.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/jabber/ui/dlgjabbereditaccountwidget.ui	2005-11-03 00:00:22.000000000 +0000
@@ -717,7 +717,4 @@
     <tabstop>leProxyJID</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgjabberregisteraccount.ui kopete-0.10.3/kopete/protocols/jabber/ui/dlgjabberregisteraccount.ui
--- kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgjabberregisteraccount.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/jabber/ui/dlgjabberregisteraccount.ui	2005-11-03 00:00:22.000000000 +0000
@@ -310,10 +310,4 @@
     <tabstop>lePasswordVerify</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>kpassdlg.h</includehint>
-    <includehint>kpassdlg.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgvcard.ui kopete-0.10.3/kopete/protocols/jabber/ui/dlgvcard.ui
--- kopete-0.10.3.orig/kopete/protocols/jabber/ui/dlgvcard.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/jabber/ui/dlgvcard.ui	2005-11-03 00:00:22.000000000 +0000
@@ -874,10 +874,4 @@
     <tabstop>teAbout</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurllabel.h</includehint>
-    <includehint>kurllabel.h</includehint>
-    <includehint>kurllabel.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/meanwhile/ui/meanwhileeditaccountbase.ui kopete-0.10.3/kopete/protocols/meanwhile/ui/meanwhileeditaccountbase.ui
--- kopete-0.10.3.orig/kopete/protocols/meanwhile/ui/meanwhileeditaccountbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/meanwhile/ui/meanwhileeditaccountbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -321,7 +321,4 @@
     <tabstop>mAutoConnect</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/msn/config/msnprefs.ui kopete-0.10.3/kopete/protocols/msn/config/msnprefs.ui
--- kopete-0.10.3.orig/kopete/protocols/msn/config/msnprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/msn/config/msnprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -210,8 +210,4 @@
     <include location="global" impldecl="in implementation">knuminput.h</include>
 </includes>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/msn/ui/msndebugrawcommand_base.ui kopete-0.10.3/kopete/protocols/msn/ui/msndebugrawcommand_base.ui
--- kopete-0.10.3.orig/kopete/protocols/msn/ui/msndebugrawcommand_base.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/msn/ui/msndebugrawcommand_base.ui	2005-11-03 00:00:22.000000000 +0000
@@ -101,7 +101,4 @@
     <tabstop>m_addNewline</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>ktextedit.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/msn/ui/msneditaccountui.ui kopete-0.10.3/kopete/protocols/msn/ui/msneditaccountui.ui
--- kopete-0.10.3.orig/kopete/protocols/msn/ui/msneditaccountui.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/msn/ui/msneditaccountui.ui	2005-11-03 00:00:22.000000000 +0000
@@ -1241,9 +1241,4 @@
     <tabstop>m_RLButton</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>knuminput.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/oscar/aim/ui/aimeditaccountui.ui kopete-0.10.3/kopete/protocols/oscar/aim/ui/aimeditaccountui.ui
--- kopete-0.10.3.orig/kopete/protocols/oscar/aim/ui/aimeditaccountui.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/oscar/aim/ui/aimeditaccountui.ui	2005-11-03 00:00:22.000000000 +0000
@@ -406,7 +406,4 @@
     <tabstop>sbxServerPort</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/oscar/aim/ui/aiminfobase.ui kopete-0.10.3/kopete/protocols/oscar/aim/ui/aiminfobase.ui
--- kopete-0.10.3.orig/kopete/protocols/oscar/aim/ui/aiminfobase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/oscar/aim/ui/aiminfobase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -240,7 +240,4 @@
     <tabstop>txtAwayMessage</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>ktextbrowser.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/oscar/icq/ui/icqadd.ui kopete-0.10.3/kopete/protocols/oscar/icq/ui/icqadd.ui
--- kopete-0.10.3.orig/kopete/protocols/oscar/icq/ui/icqadd.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/oscar/icq/ui/icqadd.ui	2005-11-03 00:00:22.000000000 +0000
@@ -125,7 +125,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/oscar/icq/ui/icqeditaccountui.ui kopete-0.10.3/kopete/protocols/oscar/icq/ui/icqeditaccountui.ui
--- kopete-0.10.3.orig/kopete/protocols/oscar/icq/ui/icqeditaccountui.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/oscar/icq/ui/icqeditaccountui.ui	2005-11-03 00:00:22.000000000 +0000
@@ -450,7 +450,4 @@
     <tabstop>chkWebAware</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kopete::ui::passwordwidget.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/oscar/icq/ui/icqsearchbase.ui kopete-0.10.3/kopete/protocols/oscar/icq/ui/icqsearchbase.ui
--- kopete-0.10.3.orig/kopete/protocols/oscar/icq/ui/icqsearchbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/oscar/icq/ui/icqsearchbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -434,12 +434,4 @@
     <tabstop>uin</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klistview.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/sms/services/smsclientprefs.ui kopete-0.10.3/kopete/protocols/sms/services/smsclientprefs.ui
--- kopete-0.10.3.orig/kopete/protocols/sms/services/smsclientprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/sms/services/smsclientprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -124,12 +124,4 @@
     </vbox>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/sms/services/smssendprefs.ui kopete-0.10.3/kopete/protocols/sms/services/smssendprefs.ui
--- kopete-0.10.3.orig/kopete/protocols/sms/services/smssendprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/sms/services/smssendprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -180,9 +180,4 @@
     <tabstop>provider</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/sms/ui/smsactprefs.ui kopete-0.10.3/kopete/protocols/sms/ui/smsactprefs.ui
--- kopete-0.10.3.orig/kopete/protocols/sms/ui/smsactprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/sms/ui/smsactprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -429,7 +429,4 @@
     <include location="global" impldecl="in implementation">knuminput.h</include>
 </includes>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>krestrictedline.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/sms/ui/smsuserprefs.ui kopete-0.10.3/kopete/protocols/sms/ui/smsuserprefs.ui
--- kopete-0.10.3.orig/kopete/protocols/sms/ui/smsuserprefs.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/sms/ui/smsuserprefs.ui	2005-11-03 00:00:22.000000000 +0000
@@ -112,7 +112,4 @@
     </vbox>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/winpopup/ui/wpaddcontactbase.ui kopete-0.10.3/kopete/protocols/winpopup/ui/wpaddcontactbase.ui
--- kopete-0.10.3.orig/kopete/protocols/winpopup/ui/wpaddcontactbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/winpopup/ui/wpaddcontactbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -181,10 +181,4 @@
     <tabstop>mRefresh</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kcombobox.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/winpopup/ui/wpeditaccountbase.ui kopete-0.10.3/kopete/protocols/winpopup/ui/wpeditaccountbase.ui
--- kopete-0.10.3.orig/kopete/protocols/winpopup/ui/wpeditaccountbase.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/winpopup/ui/wpeditaccountbase.ui	2005-11-03 00:00:22.000000000 +0000
@@ -201,7 +201,4 @@
     <slot>installSamba()</slot>
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kopete-0.10.3.orig/kopete/protocols/winpopup/ui/wpuserinfowidget.ui kopete-0.10.3/kopete/protocols/winpopup/ui/wpuserinfowidget.ui
--- kopete-0.10.3.orig/kopete/protocols/winpopup/ui/wpuserinfowidget.ui	2005-11-02 23:59:06.000000000 +0000
+++ kopete-0.10.3/kopete/protocols/winpopup/ui/wpuserinfowidget.ui	2005-11-03 00:00:22.000000000 +0000
@@ -196,10 +196,4 @@
     <tabstop>sServer</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="0"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-</includehints>
 </UI>
