--- kopete-0.12.0/kopete/plugins/cryptography/cryptographyplugin.h	2006-05-31 02:23:58.000000000 +0200
+++ kopete-0.12.0_patched/kopete/plugins/cryptography/cryptographyplugin.h	2006-06-13 23:18:49.000000000 +0200
@@ -87,6 +87,7 @@
 	bool mAlsoMyKey;
 	bool mAskPassPhrase;
 	bool mCachePassPhrase;
+	bool mShowNoCryptInfo;
 };
 
 #endif
--- kopete-0.12.0/kopete/plugins/cryptography/cryptographyplugin.cpp	2006-05-31 02:23:58.000000000 +0200
+++ kopete-0.12.0_patched/kopete/plugins/cryptography/cryptographyplugin.cpp	2006-06-13 23:33:56.000000000 +0200
@@ -108,6 +108,8 @@
 	  mCachePassPhrase = Never;
 	mCacheTime = config->readNumEntry("Cache_Time", 15);
 	mAskPassPhrase = config->readBoolEntry("No_Passphrase_Handling", false);
+	/* patch for incoming outgoing message */
+	mShowNoCryptInfo = config->readBoolEntry("No_Cryptmessage_Handling", false);
 }
 
 CryptographyPlugin* CryptographyPlugin::plugin()
@@ -184,12 +186,21 @@
 					.replace( QRegExp( QString::fromLatin1( "\\s\\s" ) ), QString::fromLatin1( "&nbsp; " ) );
 			}
 
-			msg.setBody( QString::fromLatin1("<table width=\"100%\" border=0 cellspacing=0 cellpadding=0><tr><td class=\"highlight\"><font size=\"-1\"><b>")
-				+ i18n("Outgoing Encrypted Message: ")
-				+ QString::fromLatin1("</b></font></td></tr><tr><td class=\"highlight\">")
-				+ plainBody
-				+ QString::fromLatin1(" </td></tr></table>")
-				, Kopete::Message::RichText );
+			if(!mShowNoCryptInfo){ // patch
+				msg.setBody( QString::fromLatin1("<table width=\"100%\" border=0 cellspacing=0 cellpadding=0><tr><td class=\"highlight\"><font size=\"-1\"><b>")
+					+ i18n("Outgoing Encrypted Message :")
+					+ QString::fromLatin1("</b></font></td></tr><tr><td class=\"highlight\">")
+					+ plainBody
+					+ QString::fromLatin1(" </td></tr></table>")
+					, Kopete::Message::RichText );
+			} else {
+				msg.setBody( QString::fromLatin1("<table width=\"100%\" border=0 cellspacing=0 cellpadding=0>")
+					+ QString::fromLatin1("<tr><td class=\"highlight\">")
+					+ plainBody
+					+ QString::fromLatin1(" </td></tr></table>")
+					, Kopete::Message::RichText );
+
+			}	
 		}
 
 		//if there are too messages in cache, clear the cache
@@ -215,12 +226,21 @@
 			body = Kopete::Message::escape( body );
 		}
 
-		msg.setBody( QString::fromLatin1("<table width=\"100%\" border=0 cellspacing=0 cellpadding=0><tr><td class=\"highlight\"><font size=\"-1\"><b>")
-			+ i18n("Incoming Encrypted Message: ")
-			+ QString::fromLatin1("</b></font></td></tr><tr><td class=\"highlight\">")
-			+ body
-			+ QString::fromLatin1(" </td></tr></table>")
-			, Kopete::Message::RichText );
+		if(!mShowNoCryptInfo){ // patch
+			msg.setBody( QString::fromLatin1("<table width=\"100%\" border=0 cellspacing=0 cellpadding=0><tr><td class=\"highlight\"><font size=\"-1\"><b>")
+				+ i18n("Incoming Encrypted Message: ")
+				+ QString::fromLatin1("</b></font></td></tr><tr><td class=\"highlight\">")
+				+ body
+				+ QString::fromLatin1(" </td></tr></table>")
+				, Kopete::Message::RichText );
+		} else {
+			msg.setBody( QString::fromLatin1("<table width=\"100%\" border=0 cellspacing=0 cellpadding=0>")
+				+ QString::fromLatin1("<tr><td class=\"highlight\">")
+				+ body
+				+ QString::fromLatin1(" </td></tr></table>")
+				, Kopete::Message::RichText );
+
+		}
 	}
 
 }
--- kopete-0.12.0/kopete/plugins/cryptography/cryptographyprefsbase.ui	2006-05-31 02:23:58.000000000 +0200
+++ kopete-0.12.0_patched/kopete/plugins/cryptography/cryptographyprefsbase.ui	2006-06-13 22:37:59.000000000 +0200
@@ -1,4 +1,4 @@
-<!DOCTYPE UI><UI version="3.1" stdsetdef="1">
+<!DOCTYPE UI><UI version="3.3" stdsetdef="1">
 <class>CryptographyPrefsUI</class>
 <author>Olivier Goffart</author>
 <widget class="QWidget">
@@ -10,161 +10,266 @@
             <x>0</x>
             <y>0</y>
             <width>403</width>
-            <height>287</height>
+            <height>322</height>
         </rect>
     </property>
-    <grid>
+    <property name="caption">
+        <string>CryptographyPrefsUI</string>
+    </property>
+    <widget class="QLabel">
+        <property name="name">
+            <cstring>TextLabel1</cstring>
+        </property>
+        <property name="geometry">
+            <rect>
+                <x>11</x>
+                <y>11</y>
+                <width>138</width>
+                <height>26</height>
+            </rect>
+        </property>
+        <property name="text">
+            <string>Your private PGP key:</string>
+        </property>
+    </widget>
+    <widget class="QPushButton">
+        <property name="name">
+            <cstring>m_selectOwnKey</cstring>
+        </property>
+        <property name="geometry">
+            <rect>
+                <x>309</x>
+                <y>11</y>
+                <width>83</width>
+                <height>26</height>
+            </rect>
+        </property>
+        <property name="text">
+            <string>Select...</string>
+        </property>
+    </widget>
+    <widget class="KLineEdit">
+        <property name="name">
+            <cstring>PGP_private_key</cstring>
+        </property>
+        <property name="geometry">
+            <rect>
+                <x>155</x>
+                <y>12</y>
+                <width>148</width>
+                <height>24</height>
+            </rect>
+        </property>
+        <property name="readOnly">
+            <bool>true</bool>
+        </property>
+    </widget>
+    <widget class="QCheckBox">
         <property name="name">
-            <cstring>unnamed</cstring>
+            <cstring>Also_my_key</cstring>
         </property>
-        <widget class="QLabel" row="0" column="0">
+        <property name="geometry">
+            <rect>
+                <x>11</x>
+                <y>43</y>
+                <width>381</width>
+                <height>22</height>
+            </rect>
+        </property>
+        <property name="text">
+            <string>Encrypt outgoing messages with this key</string>
+        </property>
+        <property name="whatsThis" stdset="0">
+            <string>&lt;qt&gt;Check this box if you want to encrypt outgoing messages with this key, so that you will be able to decrypt them yourself later.&lt;br&gt;
+&lt;b&gt;Warning:&lt;/b&gt; This can increase the size of messages, and some protocols will refuse to send your messages because they are too large.</string>
+        </property>
+    </widget>
+    <widget class="QButtonGroup">
+        <property name="name">
+            <cstring>m_cache</cstring>
+        </property>
+        <property name="geometry">
+            <rect>
+                <x>11</x>
+                <y>129</y>
+                <width>381</width>
+                <height>123</height>
+            </rect>
+        </property>
+        <property name="title">
+            <string>Cache Passphrase</string>
+        </property>
+        <widget class="QRadioButton">
             <property name="name">
-                <cstring>TextLabel1</cstring>
-            </property>
-            <property name="text">
-                <string>Your private PGP key:</string>
+                <cstring>Cache_Till_App_Close</cstring>
             </property>
-        </widget>
-        <widget class="QPushButton" row="0" column="2">
-            <property name="name">
-                <cstring>m_selectOwnKey</cstring>
+            <property name="geometry">
+                <rect>
+                    <x>11</x>
+                    <y>23</y>
+                    <width>359</width>
+                    <height>22</height>
+                </rect>
             </property>
             <property name="text">
-                <string>Select...</string>
-            </property>
-        </widget>
-        <widget class="KLineEdit" row="0" column="1">
-            <property name="name">
-                <cstring>PGP_private_key</cstring>
+                <string>Until Kopete closes</string>
             </property>
-            <property name="readOnly">
+            <property name="checked">
                 <bool>true</bool>
             </property>
         </widget>
-        <widget class="QCheckBox" row="1" column="0" rowspan="1" colspan="3">
+        <widget class="QSpinBox">
             <property name="name">
-                <cstring>Also_my_key</cstring>
+                <cstring>Cache_Time</cstring>
             </property>
-            <property name="text">
-                <string>Encrypt outgoing messages with this key</string>
+            <property name="geometry">
+                <rect>
+                    <x>62</x>
+                    <y>51</y>
+                    <width>151</width>
+                    <height>24</height>
+                </rect>
+            </property>
+            <property name="sizePolicy">
+                <sizepolicy>
+                    <hsizetype>7</hsizetype>
+                    <vsizetype>0</vsizetype>
+                    <horstretch>0</horstretch>
+                    <verstretch>0</verstretch>
+                </sizepolicy>
             </property>
-            <property name="whatsThis" stdset="0">
-                <string>&lt;qt&gt;Check this box if you want to encrypt outgoing messages with this key, so that you will be able to decrypt them yourself later.&lt;br&gt;
-&lt;b&gt;Warning:&lt;/b&gt; This can increase the size of messages, and some protocols will refuse to send your messages because they are too large.</string>
+            <property name="maxValue">
+                <number>999</number>
+            </property>
+            <property name="minValue">
+                <number>1</number>
+            </property>
+            <property name="value">
+                <number>15</number>
             </property>
         </widget>
-        <spacer row="5" column="1">
+        <widget class="QLabel">
             <property name="name">
-                <cstring>spacer3</cstring>
-            </property>
-            <property name="orientation">
-                <enum>Vertical</enum>
+                <cstring>textLabel1</cstring>
             </property>
-            <property name="sizeType">
-                <enum>Expanding</enum>
+            <property name="geometry">
+                <rect>
+                    <x>219</x>
+                    <y>54</y>
+                    <width>151</width>
+                    <height>17</height>
+                </rect>
+            </property>
+            <property name="sizePolicy">
+                <sizepolicy>
+                    <hsizetype>7</hsizetype>
+                    <vsizetype>0</vsizetype>
+                    <horstretch>0</horstretch>
+                    <verstretch>0</verstretch>
+                </sizepolicy>
             </property>
-            <property name="sizeHint">
-                <size>
-                    <width>21</width>
-                    <height>50</height>
-                </size>
+            <property name="text">
+                <string>minutes</string>
             </property>
-        </spacer>
-        <widget class="QButtonGroup" row="4" column="0" rowspan="1" colspan="3">
+        </widget>
+        <widget class="QRadioButton">
             <property name="name">
-                <cstring>m_cache</cstring>
+                <cstring>Cache_Till_Time</cstring>
+            </property>
+            <property name="geometry">
+                <rect>
+                    <x>11</x>
+                    <y>52</y>
+                    <width>45</width>
+                    <height>22</height>
+                </rect>
+            </property>
+            <property name="sizePolicy">
+                <sizepolicy>
+                    <hsizetype>1</hsizetype>
+                    <vsizetype>0</vsizetype>
+                    <horstretch>0</horstretch>
+                    <verstretch>0</verstretch>
+                </sizepolicy>
             </property>
-            <property name="title">
-                <string>Cache Passphrase</string>
+            <property name="text">
+                <string>For</string>
             </property>
-            <grid>
-                <property name="name">
-                    <cstring>unnamed</cstring>
-                </property>
-                <widget class="QRadioButton" row="0" column="0" rowspan="1" colspan="3">
-                    <property name="name">
-                        <cstring>Cache_Till_App_Close</cstring>
-                    </property>
-                    <property name="text">
-                        <string>Until Kopete closes</string>
-                    </property>
-                    <property name="checked">
-                        <bool>true</bool>
-                    </property>
-                </widget>
-                <widget class="QSpinBox" row="2" column="1">
-                    <property name="name">
-                        <cstring>Cache_Time</cstring>
-                    </property>
-                    <property name="sizePolicy">
-                        <sizepolicy>
-                            <hsizetype>7</hsizetype>
-                            <vsizetype>0</vsizetype>
-                            <horstretch>0</horstretch>
-                            <verstretch>0</verstretch>
-                        </sizepolicy>
-                    </property>
-                    <property name="maxValue">
-                        <number>999</number>
-                    </property>
-                    <property name="minValue">
-                        <number>1</number>
-                    </property>
-                    <property name="value">
-                        <number>15</number>
-                    </property>
-                </widget>
-                <widget class="QLabel" row="2" column="2">
-                    <property name="name">
-                        <cstring>textLabel1</cstring>
-                    </property>
-                    <property name="sizePolicy">
-                        <sizepolicy>
-                            <hsizetype>7</hsizetype>
-                            <vsizetype>0</vsizetype>
-                            <horstretch>0</horstretch>
-                            <verstretch>0</verstretch>
-                        </sizepolicy>
-                    </property>
-                    <property name="text">
-                        <string>minutes</string>
-                    </property>
-                </widget>
-                <widget class="QRadioButton" row="2" column="0">
-                    <property name="name">
-                        <cstring>Cache_Till_Time</cstring>
-                    </property>
-                    <property name="sizePolicy">
-                        <sizepolicy>
-                            <hsizetype>1</hsizetype>
-                            <vsizetype>0</vsizetype>
-                            <horstretch>0</horstretch>
-                            <verstretch>0</verstretch>
-                        </sizepolicy>
-                    </property>
-                    <property name="text">
-                        <string>For</string>
-                    </property>
-                </widget>
-                <widget class="QRadioButton" row="3" column="0" rowspan="1" colspan="3">
-                    <property name="name">
-                        <cstring>Cache_Never</cstring>
-                    </property>
-                    <property name="text">
-                        <string>Never</string>
-                    </property>
-                </widget>
-            </grid>
         </widget>
-        <widget class="QCheckBox" row="3" column="0" rowspan="1" colspan="3">
+        <widget class="QRadioButton">
             <property name="name">
-                <cstring>No_Passphrase_Handling</cstring>
+                <cstring>Cache_Never</cstring>
+            </property>
+            <property name="geometry">
+                <rect>
+                    <x>11</x>
+                    <y>81</y>
+                    <width>359</width>
+                    <height>22</height>
+                </rect>
             </property>
             <property name="text">
-                <string>Do not ask for the passphrase</string>
+                <string>Never</string>
             </property>
         </widget>
-    </grid>
+    </widget>
+    <spacer>
+        <property name="name">
+            <cstring>spacer3</cstring>
+        </property>
+        <property name="orientation">
+            <enum>Vertical</enum>
+        </property>
+        <property name="sizeType">
+            <enum>Expanding</enum>
+        </property>
+        <property name="sizeHint">
+            <size>
+                <width>21</width>
+                <height>50</height>
+            </size>
+        </property>
+        <property name="geometry">
+            <rect>
+                <x>210</x>
+                <y>260</y>
+                <width>21</width>
+                <height>50</height>
+            </rect>
+        </property>
+    </spacer>
+    <widget class="QCheckBox">
+        <property name="name">
+            <cstring>No_Passphrase_Handling</cstring>
+        </property>
+        <property name="geometry">
+            <rect>
+                <x>11</x>
+                <y>71</y>
+                <width>381</width>
+                <height>22</height>
+            </rect>
+        </property>
+        <property name="text">
+            <string>Do not ask for the passphrase</string>
+        </property>
+    </widget>
+    <widget class="QCheckBox">
+        <property name="name">
+            <cstring>No_Cryptmessage_Handling</cstring>
+        </property>
+        <property name="geometry">
+            <rect>
+                <x>11</x>
+                <y>101</y>
+                <width>381</width>
+                <height>22</height>
+            </rect>
+        </property>
+        <property name="text">
+            <string>Do not show "Incoming/Outgoing encrypted Message"</string>
+        </property>
+    </widget>
 </widget>
 <connections>
     <connection>
