diff -urN kopete-20021118.orig/kopete/protocols/yahoo/kyahoo.cpp kopete-20021118/kopete/protocols/yahoo/kyahoo.cpp
--- kopete-20021118.orig/kopete/protocols/yahoo/kyahoo.cpp	Sat Nov 16 18:43:51 2002
+++ kopete-20021118/kopete/protocols/yahoo/kyahoo.cpp	Tue Nov 26 23:05:13 2002
@@ -33,6 +33,7 @@
 #include "libyahoo2/yahoo2.h"
 #include "libyahoo2/yahoo2_callbacks.h"
 #include <iostream.h>
+#include <errno.h>
 
 /* Those gave me undefined reference errors */
 #define MAX_PREF_LEN 255
