diff -Nur kopete-orig-20021129/kopete/protocols/jabber/Makefile.am kopete-20021129/kopete/protocols/jabber/Makefile.am
--- kopete-orig-20021129/kopete/protocols/jabber/Makefile.am	Fri Nov 29 15:21:00 2002
+++ kopete-20021129/kopete/protocols/jabber/Makefile.am	Fri Jan 17 00:25:34 2003
@@ -16,7 +16,7 @@
 	ui/libkopetejabberui.la \
 	-lpsi \
 	-lssl \
-	-lqssl
+	-L/usr/X11R6/lib/qt/plugins-mt/network/
 
 data_DATA = jabber.plugin
 datadir = $(kde_datadir)/kopete
