--- kopete-20021206/kopete/protocols/jabber/Makefile.am~	Sat Dec  7 00:11:30 2002
+++ kopete-20021206/kopete/protocols/jabber/Makefile.am	Sat Dec  7 00:04:43 2002
@@ -11,7 +11,7 @@
 
 kopete_jabber_la_SOURCES = jabberprotocol.cpp jabberresource.cpp jabbercontact.cpp \
                            jabberprefs.cpp
-kopete_jabber_la_LDFLAGS = -no-undefined -module $(KDE_PLUGIN)
+kopete_jabber_la_LDFLAGS = -no-undefined -module -L$(QTDIR)/lib/qt/plugins-mt/network  $(KDE_PLUGIN)
 kopete_jabber_la_LIBADD  = ../../libkopete/libkopete.la \
 	ui/libkopetejabberui.la \
 	-lpsi \
